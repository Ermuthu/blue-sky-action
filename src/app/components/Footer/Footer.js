import React from 'react';
import PropTypes from 'prop-types';

const Footer = ({ name, version }) => (
  <footer className="border border-right-0 border-bottom-0 border-left-0">
    <div className="container-fluid">
      <div className="row">
        <div className="col-12">
          <small className="float-right">{`${name} v${version}`}</small>
        </div>
      </div>
      <div>
        <div>
          <small className="float-right">
            <a
              href="https://newsapi.org/"
              target="_blank"
              rel="noopener noreferrer"
            >
              powered by News API
            </a>
          </small>
        </div>
      </div>
    </div>
  </footer>
);

Footer.propTypes = {
  name: PropTypes.string.isRequired,
  version: PropTypes.string.isRequired
};

export default Footer;
