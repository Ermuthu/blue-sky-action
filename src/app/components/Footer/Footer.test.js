import React from 'react';
import renderer from 'react-test-renderer';
import Footer from './Footer';

test('<Footer />', () => {
  const props = {
    name: 'red-ribbon-army',
    version: '2.0.0'
  };

  const tree = renderer.create(<Footer {...props} />).toJSON();
  expect(tree).toMatchSnapshot();
});
