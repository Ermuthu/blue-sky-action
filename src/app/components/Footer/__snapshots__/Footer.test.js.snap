// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> 1`] = `
<footer
  className="border border-right-0 border-bottom-0 border-left-0"
>
  <div
    className="container-fluid"
  >
    <div
      className="row"
    >
      <div
        className="col-12"
      >
        <small
          className="float-right"
        >
          red-ribbon-army v2.0.0
        </small>
      </div>
    </div>
  </div>
</footer>
`;
